src/cljs/org/broadinstitute/firecloud_ui/page/import_data.cljs

```diff
@@ -3,17 +3,34 @@
     [clojure.string]
     [dmohs.react :as react]
     [org.broadinstitute.firecloud-ui.common.components :as comps]
+    [org.broadinstitute.firecloud-ui.common.table :as table]
     [org.broadinstitute.firecloud-ui.common.style :as style]
     [org.broadinstitute.firecloud-ui.utils :as utils]
     ))
 
 
-(defn- submit-entities [workspaceName workspaceNamespace entityFile]
+(defn- update-table [state {:keys [xhr]}]
+  (swap! state assoc :entities-loaded? true :uploadResults (utils/parse-json-string (.-responseText xhr)))
+  )
+
+
+(defn- pretend-submit-entities
+  "Insta-populates uploadResults with some sample JSON so you can just hit Upload. For testing purposes."
+  [state workspaceName workspaceNamespace entityFile]
+  (swap! state assoc :uploadResults (utils/parse-json-string "[
+  { \"entityName\" : \"foo\", \"entityType\" : \"bar\", \"success\" : false, \"message\" : \"ohno\" },
+  { \"entityName\" : \"bar\", \"entityType\" : \"baz\", \"success\" : true, \"message\" : \"hooray\" }
+  ]")
+    :entities-loaded? true)
+  )
```

Comment:
Nitpick: Indentation in this function is clunky.  I don't know if there's a rule for multi-line string literals but it's hard to see what's going on.  Maybe put :uploadResults on a new line and scoot the string literal lines over so the map keys align vertically.  Also don't give the final close paren its own line.